am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -263,6 +246,47 @@ public FilteredResourceEnvelope filterResource(@NotBlank String userId,
         return permissionsService.merge(permissionsForRoles);
     }
 
+    /**
+     * Retrieves {@link RolePermissions} filtered by role security classification for a specific resource and role.
+     *
+     * @param resourceDefinition {@link ResourceDefinition} a unique service name, resource type and resource name
+     * @param roleName           user role name
+     * @return {@link RolePermissions} a combination of permissions and security classifications for a role name
+     * @throws PersistenceException if any persistence errors were encountered
+     */
+    @AuditLog("returned role access to resource defined as '{{resourceDefinition.serviceName}}|"
+        + "{{resourceDefinition.resourceType}}|{{resourceDefinition.resourceName}}' for role '{{roleName}}': "
+        + "{{result}}")
```

Comment:
@swalker125 can you paste exemplar audit log? I wonder how `{{result}}` is rendered.

Reply:
2019-04-29T06:55:50.354 INFO  [main] u.g.h.reform.amlib.internal.aspects.AuditingAspect [Access Management audit]: returned role access to resource defined as 'CMC|Case|Claim' for role 'ec02af28-982d-4e1a-a4e8-f460e2ac7615': RolePermissions(permissions={/child=[READ], /parent/age=[READ]}, securityClassifications={/child=PUBLIC, /parent/age=PUBLIC}, roleSecurityClassification=PUBLIC, roleAccessType=ROLE_BASED)

Reply:
I guess this would be better split as result.permissions etc?

Reply:
maybe only result.permissions is important actually... Shall discuss this morning when we're in the office.

Reply:
I think it would be better to have custom format instead of relying on RolePermissions#toString. Please propose something.